### ISSUE TITLE
A11y : Cannot interact with log lines via keyboard

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46478]
**Steps to reproduce**:

Can’t Focus / Interact with log lines using keyboard-only navigation

**Actual Result**:

Degraded experience - log lines cannot be expanded to check log labels and detected fields

**Expected Result**

Elements should be more accessible to users who rely on keyboard navigation

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by Thiagomn1** [comment]
Hello @zuchka, is this issue still available? Are these the log lines you refer about?
![image](https://user-images.githubusercontent.com/85890604/195677978-68008729-7df8-4802-a638-30637fa58881.png)
